source/except.d.ts

```diff
@@ -0,0 +1,17 @@
+/**
+Create a type from an object type without certain keys.
+
+@example
+```
+import {Except} from 'type-fest';
+
+type Foo = {
+	a: number;
+	b: string;
+};
+
+type FooWithoutA = Except<Foo, 'a'>;
```

Comment:
Can you make the example exclude two props to make it clear that it works with multiple props. That was not clear to me (https://github.com/sindresorhus/type-fest/issues/27#issuecomment-490267194).